contracts/script/Deploy.s.sol

```diff
@@ -21,17 +22,34 @@ contract DeployScript is Script {
 
         address gateway = 0x6e4f1e9eA315EBFd69d18C2DB974EEf6105FB803;
 
-        VectorX lightClient = new VectorX();
+        bytes32 CREATE2_SALT = blockhash(block.number - 1);
```

Comment:
nit: i'd recommend reading it from env instead (`vm.envBytes32("CREATE2_SALT")`) as I think block.number is fixed in scripts